domain-models-runtime/src/main/java/org/folio/rest/persist/ddlgen/SchemaMaker.java

```diff
@@ -114,15 +114,13 @@ public String generateDDL(boolean recreateIndexMode) throws IOException, Templat
       int size = tables.size();
       for (int i = 0; i < size; i++) {
         Table t = tables.get(i);
-
-        if(t.getMode() == null){
+        if(t.getMode() == null){ 
```

Comment:
A space was added after the {
The editor/IDE should be configured to automatically remove trailing whitespace in java files, see https://dev.folio.org/guides/developer-setup/#coding-style

Reply:
I looked into why my ide didnt catch this, and came to the conclusion that the eclipse version of the editor config plug in does not support many key features of editor config.  This is less then ideal.  I may look into switching to VS Code or something which I use in other cases but was not able to get junit/maven working to my satisfaction when i started working with RMB.  May give it another shot.